dev/ScrollBar/ScrollBar_themeresources.xaml

```diff
@@ -420,9 +424,9 @@
                             </ControlTemplate>
                             <ControlTemplate x:Key="VerticalThumbTemplate" TargetType="Thumb">
                                 <Rectangle x:Name="ThumbVisual" Fill="{TemplateBinding Background}"
-                                           contract7Present:RadiusX="{Binding CornerRadius, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource TopLeftCornerRadiusDoubleValueConverter}}"
```

Comment:
>TopLeftCornerRadiusDoubleValueConverter [](start = 167, length = 39)

Does this resource need to be deleted somewhere?

Reply:
It's still being used in some other places so we need to keep it.